lgc/state/PalMetadata.cpp

```diff
@@ -147,8 +147,8 @@ void PalMetadata::initialize() {
   // Pre-find (or create) heavily used nodes.
   m_pipelineNode =
       m_document->getRoot().getMap(true)[Util::Abi::PalCodeObjectMetadataKey::Pipelines].getArray(true)[0].getMap(true);
-  if (!m_useRegisterFieldFormat)
-    m_registers = m_pipelineNode[".registers"].getMap(true);
+  // Add ".registers" unconditionally is a temporary workaround since PAL needs read this section for parsing.
+  m_registers = m_pipelineNode[".registers"].getMap(true);
```

Comment:
Do we actually need this change on gpuopen?

Reply:
This workaround has been removed internally; But the promotion release happens before that internal change. The lgc lit testing already fail for this metadata difference. I think we need to include the internal change to remove the workaround here.

Reply:
This workaround is not needed any more.